setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     psutil>=5.7.2
     numpy>=1.16
     astropy>=5.0.4
+    stcal>=1.3.2,<1.4
```

Comment:
Hm. I didn't realize stcal becomes a dependence. It works for now and if necessary it's easy to break this dependence.

Reply:
Looks like there is nothing stopping me from moving the the two modules from `stcal` to here (they are not used by Roman). I have looked closely at the best way to do this.

So that I don't have to rework this PR and #113 again, I think the best bet is to "temporally" accept this dependency as part of this PR (and #113), then come back and move the few bits out of `stcal` we need as a new PR.

The main reason I want to do it this way is I want to maintain continuity of the changes from `jwst` to here. Namely, I include the hash of the exact commit I copied from `jwst` to here so that when following the history one can switch repos as they follow it backwards. I would rather not have to fiddle with the history in this PR to handle moving the `stcal` bits first, then updating jwst (which will require adjustments to the datamodels), and then use the new starting point for where I copy the datamodels from.